src/express/middlewares/auth.js

```diff
@@ -108,7 +111,7 @@ function authMiddleware(namespace, options) {
             if (authorizationRequired) {
               next(new Error(errorMessage))
```

Comment:
`logger.error(errorMessage)` here ?

Reply:
I decide to remove `logger.error(errorMessage)` here since `src/express/server.js .__applyCustomErrorHandler` will log error there.